app.js

```diff
@@ -12,7 +12,7 @@ Answer = require('./models/answers');
 Question = require('./models/questions');
 
 // connect mongoose 
-mongoose.connect('mongodb://localhost/geek-tools');
+mongoose.connect('mongodb://geektoolsuser:SimCityNo3@ds155424.mlab.com:55424/geektools');
 
```

Comment:
How does Mongoose Connect link to Mongo info?

Reply:
mongoose.connect() is a function with the parameters of a uri string containing a username and password in order to connect to the after stated db